config/laravel-backup.php

```diff
@@ -49,6 +49,11 @@
         'destination' => [
 
             /*
+             * The file name used for the backup zip file.
+             */
+            'filename' => \Carbon\Carbon::now()->format('Y-m-d-H-i-s'),
```

Comment:
Doesn't this just set an invariant filename based on the current date and time, i.e. all backups would have the same filename? That's why the tests require manually overriding of the attribute.

Perhaps this should instead be a date format string which then gets instantiated for each backup.